.ci/common.sh

```diff
@@ -375,6 +365,85 @@ function hashicorp_release() {
     export AWS_SECRET_ACCESS_KEY="${okey}"
 }
 
+# Build and release project gem to RubyGems
+function publish_to_rubygems() {
+    if [ -z "${RUBYGEMS_API_KEY}" ]; then
+        fail "RUBYGEMS_API_KEY is currently unset"
+    fi
+
+    gem_config="$(mktemp -p ./)" || fail "Failed to create temporary credential file"
+    wrap gem build *.gemspec \
+         "Failed to build RubyGem"
+    printf -- "---\n:rubygems_api_key: ${RUBYGEMS_API_KEY}\n" > "${gem_config}"
+    wrap_raw gem push --config-file "${gem_config}" *.gem
+    result=$?
+    rm -f "${gem_config}"
+
+    if [ $result -ne 0 ]; then
+        fail "Failed to publish RubyGem"
+    fi
+}
+
+# Publish gem to the hashigems repository
+#
+# $1: Path to gem file to publish
+function publish_to_hashigems() {
+    path="${1}"
+    if [ -z "${path}" ]; then
+        fail "Path to built gem required for publishing to hashigems"
+    fi
+
+    # Install dependency if required
+    gem list reaper-man --installed > "${output}"
+    if [ $? -ne 0 ]; then
+        wrap_stream gem install reaper-man --no-document \
+                    "Failed to install dependency for hashigem generation"
+    fi
+
+    # Create a temporary directory to work from
+    tmpdir="$(mktemp -p ./)" ||
```

Comment:
I believe this needs a `-d` flag.

Reply:
Updating to the latest version includes this fix